feconf.py

```diff
@@ -366,6 +366,9 @@ def get_empty_ratings():
 # Disables all the new structures' pages, till they are completed.
 ENABLE_NEW_STRUCTURES = False
 
+# No. of questions to be displayed on a page in the topic editor.
+NO_OF_QUESTIONS_DISPLAYED_IN_A_PAGE = 10
```

Comment:
NUM_QUESTIONS_PER_PAGE or NUMBER_OF_QUESTIONS_PER_PAGE

(Suggest not using "NO" as an abbreviation for "NUMBER". It's ambiguous.)

Reply:
Done.